src/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_AnyAttribute.cs

```diff
@@ -0,0 +1,202 @@
+﻿using Xunit;
+using Xunit.Abstractions;
+using System.IO;
+using System.Xml.Schema;
+
+namespace System.Xml.Tests
+{
+    public class TC_SchemaSet_AnyAttribute : TC_SchemaSetBase
+    {
+        private ITestOutputHelper _output;
+        public bool bWarningCallback;
+        public bool bErrorCallback;
+        public int errorCount;
+        public int warningCount;
+        public bool WarningInnerExceptionSet = false;
+        public bool ErrorInnerExceptionSet = false;
+
+        public TC_SchemaSet_AnyAttribute(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        public void Initialize()
+        {
+            bWarningCallback = bErrorCallback = false;
+            errorCount = warningCount = 0;
+            WarningInnerExceptionSet = ErrorInnerExceptionSet = false;
+        }
```

Comment:
This Initialize method is unnecessary.  It's only ever called at the beginning of a test, so these fields will already have their default values.  A new test class is created for individual test, e.g. each theory inline data is run on its own class instance.